scripts/submit-preview-comment

```diff
@@ -35,15 +35,8 @@ fi
 echo -n "Full site preview..."
 git rev-parse --abbrev-ref HEAD | grep -q -e '^release/'
 if [ $? -eq 0 ]; then
-    git diff HEAD..origin/master --quiet -- docs
-    if [ $? -eq 0 ]; then
-        echo "ERROR"
-        echo "New release detected, but docs were not updated."
```

Comment:
🤔 this was a cool check and proved useful a few times. what will we do now?

Reply:
We actually generate docs after publishing, because the publish step is the one that bumps versions (and allows us to display the most recent versions in the docs app)

Reply:
ok cool